capstone/src/main/java/com/google/sps/servlets/announcements/ScheduleAnnouncementServlet.java

```diff
@@ -0,0 +1,90 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example club content */
+@WebServlet("/schedule-announcement")
+public class ScheduleAnnouncementServlet extends HttpServlet {
+
+  private ZoneId timeZone = ZoneId.of(Constants.TIME_ZONE);
+  private Clock clock = Clock.system(timeZone);
+
+  protected void setClock(Clock clock) {
+    this.clock = clock;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String userEmail = request.getUserPrincipal().getName();
+    final String clubName = request.getParameter(Constants.PROPERTY_NAME);
+    final String announcementContent = request.getParameter(Constants.CONTENT_PROP);
+    String scheduledDate = request.getParameter(Constants.SCHEDULED_DATE_PROP);
```

Comment:
Do users only specify the day they want to schedule not time?
It seems like day only though I'm not sure if I'm missing anything.

If they can choose a specific time then we should make sure that the timezones of the person posting and what we're saving "America/Los Angeles" is properly converted.